website/static/js/profile.js

```diff
@@ -664,28 +750,62 @@ ListViewModel.prototype.unserialize = function(data) {
 };
 
 ListViewModel.prototype.serialize = function() {
+    var self = this;
     var contents = [];
-    if (this.contents().length !== 0 && typeof(this.contents()[0].serialize() !== undefined)) {
-        for (var i=0; i < this.contents().length; i++) {
-            contents.push(this.contents()[i].serialize());
+    
+    if (self.contents().length !== 0 && typeof(self.contents()[0].serialize() !== undefined)) {
+        for (var i=0; i < self.contents().length; i++) {            
+            // If the requiredField is empty, it will not save it and will delete the blank structure from the database.  
+            if (!self.contents()[i].institutionEmpty() || !self.hasMultiple()) {
```

Comment:
This seems a bit harder to reason about than it needs to be. Consider refactoring to the following:

```javascript
if (!( self.contents()[i].institutionEmpty() && self.hasMultiple() )) {
```

This is completely preference, and is at your discretion.